Lib/multiprocessing/managers.py

```diff
@@ -1155,7 +1155,7 @@ def set(self, value):
     '__add__', '__contains__', '__delitem__', '__getitem__', '__len__',
     '__mul__', '__reversed__', '__rmul__', '__setitem__',
     'append', 'count', 'extend', 'index', 'insert', 'pop', 'remove',
-    'reverse', 'sort', '__imul__'
+    'reverse', 'sort', '__imul__', 'clear', 'copy',
```

Comment:
Wouldn't it be better to insert these two new strings between 'append' and 'count', to respect alphabetical order?